### DISCUSSION TITLE
Dataset for training Japanese language  | 用于训练日语语言的数据集？

### DISCUSSION THREAD

**Discussion by leviethung2103** [PaddlePaddle/PaddleOCR#12245] [Q&A]
Hello, 

I have some questions about Japanese paddle ocr. 

1. What are the recommended pre-trained models for training a Japanese language model?
E.g: Choose the chinese pretrained model or we can try with any language pretrained model 
2. Where can I find the Japanese text datasets that author have used to train the japa_paddle_v3 version ? 

Thank you in advance.

你好，

我有一些关于日文 Paddle OCR 的问题。

有关训练日文语言模型的推荐的预训练模型是什么？
例如：选择中文预训练模型，或者我们可以尝试使用任何其他语言的预训练模型吗？
我可以在哪里找到作者用于训练 japa_paddle_v3 版本的日文文本数据集？
提前谢谢你的帮助。

**Comment by madmalkav** [comment]
I would also like to know.

**Comment by phamkhactu** [comment]
@leviethung2103  any update?
